common/rdr/Exception.cxx

```diff
@@ -49,6 +52,24 @@ Exception::Exception(const char *format, ...) {
 	va_end(ap);
 }
 
+NetworkException::NetworkException(const char* s, int err)
+  : Exception("%s", s)
+{
+  strncat(str_, ": ", len-1-strlen(str_));
+#ifdef _WIN32
+  wchar_t *currStr = new wchar_t[len-strlen(str_)];
+  currStr = gai_strerrorW(err);
```

Comment:
This looks incorrect. If you are getting a pointer from `gai_strerrorW()` then you shouldn't allocate one yourself (it will just get lost here).